dexter/src/main/java/com/karumi/dexter/AndroidPermissionService.java

```diff
@@ -47,6 +47,9 @@ void requestPermissions(@NonNull Activity activity, @NonNull String[] permission
    */
   boolean shouldShowRequestPermissionRationale(@NonNull Activity activity,
       @NonNull String permission) {
+    if (activity == null) {
```

Comment:
Remove the `@NonNull` annotation, it was just a c&p from the original method (or move the check out of this class)